browser/src/control/jsdialog/Util.KeyboardListNavigation.ts

```diff
@@ -52,6 +43,13 @@ function moveToFocusableEntry(
 		}
 	};
 
+	const updateAriaActiveDescendant = (elem: HTMLElement) => {
```

Comment:
can't we have it inside listbox/combobox widget code?
on focus lost or something similar to treeview